tests/test_commands.py

```diff
@@ -166,10 +166,21 @@ def teardown():
         assert r.acl_setuser(username, enabled=True,
                              remove_passwords=['pass2'])
         assert len(r.acl_getuser(username)['passwords']) == 1
-
-    @skip_if_server_version_lt('6.0.0')
-    def test_acl_list(self, r, request):
-        username = 'redis-py-user'
+        # Resets and tests hash value "variable HP" set properly.
+        HP = '5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8'
+        assert r.acl_setuser(username, enabled=True, reset=True,
+                             add_passwords=[HP])
+        assert acl['passwords'] == [HP]
```

Comment:
I don't understand how the `acl` dict is updated here to include the `HP` hash. Did you forget to call `acl_getuser(...)` after the most recent `acl_setuser()`?

Or is `HP` the hash of "pass1" and it's lingering around from the previous `acl_getuser()`?

Reply:
HP is the SHA-256 hash of the word 'password'. Since Redis will now respect hashes as entered passwords. What this is doing is defining the variable to use in order to pass in the password and retesting that the hash remains a hash and does not transform into another like it would if you used 'password' in the password instead.